content/water-data-science-2021.md

```diff
@@ -0,0 +1,94 @@
+</p>
+</div>
+We’re focusing heavily in 2021 on a subset of KGML - which we’re calling Process-Guided Deep Learning - to make improved predictions of water quality or quantity that are guided by physical models and laws (such as conservation of energy, see <a href="https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2019WR024922">here</a>). These are advanced hybrid models that take the best of what our field has to offer for physics-based models and the best machine learning frameworks for spatiotemporally-aware models. The results of these collaborations have been exciting, with large improvements in our predictive accuracy and ability to transfer these models into unseen conditions and still make good predictions. We are currently working on the prediction of water temperature in lakes and streams with these methods, and also working on similar concepts for predicting stream discharge. The place for machine learning in USGS will continue to grow in the next several years and KGML and PGDL will generate some very interesting conversations regarding environmental modeling architectures. 
+
+
+Forecasting changes to water resources {#data-assimilation}
+--------------------
+
+<div class="grid-row">
+{{< figure src="/static/FY21DataSci/sequential_DA.png" alt="Sequential data assimilation forecasting" class="side-by-side" >}}
+
+<p class="side-by-side" >
+Continuing on with the theme of building better predictions with data, we are leading a new project to forecast stream and reservoir temperatures in the Delaware River Basin using data assimilation and machine learning. Data assimilation (DA) is a technique that makes iterative improvements to forecasts by weighting the confidence in new observations (e.g. streamflow and water temperature) with the confidence in model predictions. These improvements or adjustments can include changes to parameters and/or state variables that are internal to the model. DA is an especially powerful tool for environmental forecasts because it can take advantage of the continued accumulation of data to improve predictions and leverage real-time observations for updates, such as those from the thousands of USGS gages that pepper the landscape. 
+</p>
+This technique is clearly data-intensive and requires thoughtful automation to perform in a forecast setting. We are able to leverage several of the other Data Science capabilities to strengthen his effort, such as reproducible workflows and data visualization. Forecasting is relatively new but important territory for USGS, and we’re very pleased to be part of leading the simultaneous growth in forecasting and DA in the Water Resources Mission Area. 
+</div>
+
+Capturing reproducible and efficient data workflows {#reproducibility}
+--------------------
+![USGS data science reproducibility](/static/FY21DataSci/reproducibility.png)
```

Comment:
please use the figure short code like above, please add captions and descriptive alt attributes as well.